source/index.html.md

```diff
@@ -207,6 +199,60 @@ This endpoint creates a subscription record under a business account
 | `subscription`       | subscription|
 | *body*               |             |
 
+
+### Subscription policy
+
+> Example Subscription body for a tier whose policy value is **SUBSCRIPTION**
+
+```json
+{
+  "customerId": "9e0fb8d9-f7bf-4db0-931b-3e5a04b49c68",
+  "tierId": "2fb2a2d4-2454-4a06-8b4b-6fd7c74dd7dc",
+  "amount": 20,
+  "startDate": "2022-02-25"
+}
+```
+
+This endpoint creates a subscription record under a business account.
```

Comment:
Remove the sentence "This endpoint creates a subscription record under a business account." You are repeating what's already stated above.